Mergin/create_project_wizard.py

```diff
@@ -493,9 +494,16 @@ def accept(self):
             new_proj.write()
             reload_project = True
 
+            # copy datum shift grids
+            package_datum_grids(os.path.join(self.project_dir, "proj"))
+
         elif self.init_page.cur_proj_no_pack_btn.isChecked():
             cur_proj = QgsProject.instance()
             cur_proj.write()
+
+            # copy datum shift grids
+            package_datum_grids(os.path.join(QgsProject.instance().homePath(), "proj"))
```

Comment:
why not like this?
```suggestion
            package_datum_grids(os.path.join(self.project_dir, "proj"))
```
The problem is that `homePath()` can be arbitrary user-defined directory...